doc/new-book/routing.adoc

```diff
@@ -575,6 +575,20 @@ connector {
 For information about additional attributes, see link:{qdrouterdConfManPageUrl}#_connector[connector] in the `qdrouterd.conf` man page.
 --
 
+. If you want clients to send local transactions to the broker, create a link route for the transaction coordinator:
++
+--
+[options="nowrap",subs="+quotes"]
+----
+linkRoute {
+    prefix: $coordinator  <1>
+    connection: __CONNECTOR_NAME__
+    direction: in
+}
+----
+<1> The `$coordinator` prefix designates this link route as a transaction coordinator. When the client opens a transacted session, the transactional state is propagated along this link route to the broker.
```

Comment:
The coordinator link is used for declaring and completing (committing or rolling back) the transactions. In the AMQP spec, the term 'transactional state' is used to tie the publication and consumption of messages to a given transaction, and is part of the transfer or disposition - loosely the message or its acknowledgement - which do not go over the coordinator link.

Perhaps it would be less (potentially) confusing to change from 'the transactional state is propagated along this link route to the broker' to 'requests to start or end a transaction are propagated along this link route to the broker.' ??